doc/specifications/proposed/named_queries/named_queries.md

```diff
@@ -0,0 +1,53 @@
+## Overview
+
+Named queries are Content or Location Query objects that are predefined in the system.
+
+They can be used in multiple ways:
+- from twig templates, using a sub-request: {{ render( controller( ez_query, {'identifier': 'latest_articles' ) ) }}
+- from php code, by getting the Query from a service
+- from REST, by executing a predefined view / query
+
+They
+## Components
+
+### REST
+
+All REST calls on `/content/views`.
+
+### Query controller
+```twig
+{{ render( controller( ez_query, {'identifier': 'latest_articles', 'parameters': {'type': 'article'}} ) ) }}
+```
+
+### Services
+
+A main `QueryService` / `ezplatform.query_service`, with methods to load/create/update/delete named queries.
```

Comment:
We don't have persisted queries yet, and they're not a high priority. The idea behind named queries is to standardize "content listing" in general.

Yes, queries can be added as flat files already, and it is an implementation detail. What we add on top of this is:
- Standardization of those query files (I'm considering auto-registration based on the query's path, like Sf2 controllers)
- Template of query results (just pushed a commit that introduces this)

The goal is to fix, at the same time, the "list" controller people have been missing for a while.